infra.bs

```diff
@@ -299,10 +301,12 @@ contains, in the range 0x61 (a) to 0x7A (z), inclusive, by 0x20.
 <h3 id=code-points>Code points</h3>
 
 <p>A <dfn export lt="code point|character">code point</dfn> is a Unicode code point and is
-represented as a four-to-six digit hexadecimal number, typically prefixed with "U+". A
-<a>code point</a> may be followed by its name, by its rendered form between parentheses, or both.
-Documents using the Infra Standard are encouraged to follow <a>code points</a> by their name when
-they cannot be rendered, and rendered form between parentheses otherwise, for legibility.
+represented as a four-to-six digit hexadecimal number, typically prefixed with "U+".
+
+<p>A <a>code point</a> may be followed by its name, by its rendered form between parentheses when it
+is not U+0028 or U+0029, or both. Documents using the Infra Standard are encouraged to follow
```

Comment:
This sentence has now become a bit ambiguous as it's grown, so I'd say "or by both"